modules/obam/manifests/params.pp

```diff
@@ -16,31 +16,21 @@
 
 class obam::params inherits ob_common::params{
 
-  $start_script_template  = 'bin/wso2server.sh'
+  $start_script_template  = 'bin/api-manager.sh'
 
   # Maximum and minimum memory sizes for JVM
   $jvmxms = '2048m'
   $jvmxmx = '2048m'
 
-  # jaggeryapps/admin/site/conf/site.json params
-  $work_flow_server_url = "https://${bps_hostname}:9445/services/"
-
   # Deployment.toml file
```

Comment:
As per https://github.com/wso2/puppet-open-banking/pull/40/files#r660332994, we can use only one variable to declare the toml file path as same as above 'start_script_template'. Check in obiam scripts also.

Reply:
Done, Please check last commit. Thanks.